pycbc/waveform/compress.py

```diff
@@ -685,8 +738,12 @@ def write_to_hdf(self, fp, template_hash, root=None):
         else:
             root = '%s/'%(root)
         group = '%scompressed_waveforms/%s' %(root, str(template_hash))
-        for param in ['amplitude', 'phase', 'sample_points']:
-            fp['%s/%s' %(group, param)] = self._get(param)
+        if self.interpolation == "linear":
+           for param in ['amplitude', 'phase', 'phase_freq_points']:
```

Comment:
__[flake8]__
* [E111] indentation is not a multiple of four

<!-- Comment by [SideCI](https://sideci.com) -->